iceoryx_utils/include/iceoryx_utils/cxx/optional.hpp

```diff
@@ -124,7 +125,7 @@ class optional
     /// @param[in] value value to assign to the underlying optional value
     /// @return reference to the current optional
     template <typename U = T>
-    typename ::std::enable_if<!::std::is_same<U, optional<T>&>::value, optional>::type& operator=(U&& value) noexcept;
+    typename std::enable_if<!std::is_same<U, optional<T>&>::value, optional>::type& operator=(U&& value) noexcept;
```

Comment:
@elfenpiff is this correct or a merge error?

Reply:
In all other code references it's `std::` and not `::std::`.